app/controllers/provider/invitee_signups_controller.rb

```diff
@@ -49,7 +49,8 @@ def can_create?
   end
 
   def build_user
-    @user = @invitation.make_user(params[:user] || {})
+    permitted_params = params.permit(user: %i[password username])
+    @user = @invitation.make_user(permitted_params[:user] || {})
```

Comment:
```suggestion
    @user = @invitation.make_user(permitted_params[:user])
```
No need of `|| {}` since `#permit` always return an `ActionController::Parameters`